### PR TITLE
Adds a set provider class

### PR DESCRIPTION
**Pull request by peterfox**
# Changes

- Adds a Set Provider to the project

# Why

This is now the standard for Rector repositories. Having this Set Provider will make it easier to display them on [GetRector.com/](https://getrector.com/find-rule)

### CODE REVIEW

**Comment by peterfox** [comment]
@TomasVotruba if you wouldn't mind taking a look and letting me know if I need to make further changes.

**Comment by TomasVotruba** [comment]
Looks good :+1:  Let's ship it

File: src/Set/LaravelSetProvider.php [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+            new Set(self::GROUP_NAME, 'Code quality', LaravelSetList::LARAVEL_CODE_QUALITY),
+            new Set(
+                self::GROUP_NAME,
+                'Container strings to FQN types',
+                LaravelSetList::LARAVEL_CONTAINER_STRING_TO_FULLY_QUALIFIED_NAME,
+            ),
+            new Set(
+                self::GROUP_NAME,
+                'Code Quality for Laravel',
+                LaravelSetList::LARAVEL_CODE_QUALITY,
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply src/Set/LaravelSetProvider.php:68]: @GeniJaho You're right. Indeed, there should be just one unique items :+1:

[the remark this answers is not in the document]
Comment (peterfox) [reply src/Set/LaravelSetProvider.php:68]: I'll fix this and add a quick unit test 😓

[the remark this answers is not in the document]
Comment (peterfox) [reply src/Set/LaravelSetProvider.php:68]: Created a PR for the fix https://github.com/driftingly/rector-laravel/pull/249

**Pull request merged**